gateway-provider-security-authz-composite/src/test/java/org/apache/knox/gateway/deploy/impl/CompositeAuthzProviderTest.java

```diff
@@ -83,13 +85,19 @@ public void testingNullandEmptyProviderNames() throws Exception {
     String testnames = "";
     CompositeAuthzDeploymentContributor c = new CompositeAuthzDeploymentContributor();
     List providerNames = c.parseProviderNames(testnames);
-    assertEquals(providerNames.size(), 1);
-    assertEquals(providerNames.get(0), "");
+    assertSame(providerNames.size(),0);
```

Comment:
Here the order of parameters are reversed. The first is the expected and the second is the actual. 
This is not a correctness issue, but the error message will misleading if the test fails.

For example if providerNames has a size of 3 it'll say: expected 3 got 0, but it is the other way around.

More importantly what's the reason for using assertSame instead of assertEquals? My understanding is that assertSame checks identity (`==`) not equality (`.equals`).